src/device.rs

```diff
@@ -221,38 +226,41 @@ pub struct CAdapterInfo {
     pub backend: Backend,
 }
 
+#[allow(non_snake_case)]
 #[repr(C)]
-pub struct CDeviceDescriptor {
+pub struct CDeviceDescriptor<'c> {
```

Comment:
do we really need the `C` prefixes for these types? They are not colliding with `wgt::DeviceDescriptor`, since we aren't glob-importing `wgt`, so we could call them the same.

Reply:
See also - #78 as a radical solution to this